packages/main/src/SegmentedButton.ts

```diff
@@ -63,14 +65,19 @@ type SegmentedButtonSelectionChangeEventDetail = {
  * Fired when the selected item changes.
  *
  * @event sap.ui.webc.main.SegmentedButton#selection-change
+ * @deprecated As of 1.14.0. This parameter is not guaranteed in future releases.
```

Comment:
Did you check how this is shown up, because we don'd deprecate the event, just a single event param. Maybe we can see how it's in OpenUI5, as in the ui5webc project, this is the first time we deprecate an event param.

Reply:
The current place for the deprecated tag seems to not be correct. If the position is changed line in the OpenUI5 project however no information gets generated for the deprecation at all.

Reply:
Documentation is adjusted as per what we've discussed.